managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+      "Resource": "docker://docker.io/agrare/clone-template:latest",
+      "Next": "CheckTaskComplete",
+      "Credentials": {
+        "api_user.$": "$.api_user",
+        "api_password.$": "$.api_password",
+        "vcenter_user.$": "$.vcenter_user",
+        "vcenter_password.$": "$.vcenter_password"
+      },
+      "Parameters": {
+        "PROVIDER_ID.$": "$.dialog_provider",
+        "VERIFY_SSL": false,
+        "TEMPLATE.$": "$.dialog_source_template",
+        "NAME.$": "$.dialog_vm_name"
+      }
+    },
+
+    "CheckTaskComplete": {
+      "Type": "Task",
+      "Resource": "docker://docker.io/agrare/check-task-complete:latest",
+      "Next": "PollTaskComplete",
+      "Credentials": {
+        "vcenter_user.$": "$.vcenter_user",
+        "vcenter_password.$": "$.vcenter_password"
+      },
+      "Parameters": {
+        "VCENTER_HOST.$": "$.vcenter_host",
+        "TASK.$": "$.task"
+      }
+    },
+
+    "PollTaskComplete": {
+      "Type": "Choice",
+      "Choices": [
+        {
+          "Variable": "$.state",
+          "StringEquals": "success",
+          "Next": "PowerOnVM"
+        },
+        {
+          "Variable": "$.state",
+          "StringEquals": "running",
+          "Next": "RetryState"
+        },
+        {
+          "Variable": "$.state",
+          "StringEquals": "error",
+          "Next": "FailState"
+        }
+      ],
+      "Default": "FailState"
+    },
+
+    "RetryState": {
+      "Type": "Wait",
+      "Seconds": 5,
+      "Next": "CheckTaskComplete"
+    },
+
+    "PowerOnVM": {
+      "Type": "Task",
+      "Resource": "docker://docker.io/agrare/power-on-vm:latest",
+      "Next": "SuccessState",
+      "Credentials": {
+        "vcenter_user.$": "$.vcenter_user",
+        "vcenter_password.$": "$.vcenter_password"
+      },
+      "Parameters": {
+        "VCENTER_HOST.$": "$.vcenter_host",
+        "VM.$": "$.vm"
+      }
+    },
+
+    "FailState": {
+      "Type": "Fail",
+      "Error": "FailStateError",
+      "Cause": "No Matches!"
+    },
+
+    "SuccessState": {
+      "Type": "Succeed"
+    }
+  }
+}
+```
+
+### Viewing workflow details
+
+When workflows are available, you can view the details for each workflow, such as to verify the payload. To view the details for a workflow:
+
+1. Click **Automation** > **Embedded Workflows**. Then, click **Workflows** to see the workflows list.
+
+2. Click a workflow. The default *Text* view of the workflow opens where you see the code content.
+
+   ![Text view](../images/embedworkflow_textview.png){: caption="Text view" caption-side="bottom"}
+
+   Alternatively, click the **Graph** tab to see a graph of the same data.
+
+   ![Graph view](../images/embedworkflow_graphView.png){: caption="Graph view" caption-side="bottom"}
+
+3. If your workflow has credential fields, then create a workflow credential for each of the fields. Then, map the workflow credential to the workflow.
+   <!-- Add substeps from development - A.G to provide.-->
```

Comment:
I think we can reference the Credentials section here saying that you need to create a Workflow Credential, then here in this section we can cover the `Map Credentials` UI

Reply:
I have updated this section to reference/link to the adding credentials section and added a new step for mapping the credentials. This content will be added later.